src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -119,24 +139,61 @@ function _assignPanelId(element) {
   }
 }
 
+/*
+ * Triggers
+ */
+
+function _parseTrigger(element) {
```

Comment:
I think it would also be great if we could leave come comments to give future developers some context as to why we are doing these transformations.